### PR TITLE
Upgrade to v1.15.0 of the NewRelic Terraform Provider

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: resources.go [hunk 1]
```diff
@@ -150,30 +150,44 @@ func Provider() tfbridge.ProviderInfo {
...
+			"newrelic_alert_policy_channel":         {Tok: makeResource(mainMod, "AlertPolicyChannel")},
+			"newrelic_dashboard":                    {Tok: makeResource(mainMod, "Dashboard")},
+			"newrelic_infra_alert_condition":        {Tok: makeResource(mainMod, "InfraAlertCondition")},
+			"newrelic_nrql_alert_condition":         {Tok: makeResource(mainMod, "NrqlAlertCondition")},
+			"newrelic_synthetics_alert_condition":   {Tok: makeResource(syntheticsMod, "AlertCondition")},
+			"newrelic_synthetics_monitor":           {Tok: makeResource(syntheticsMod, "Monitor")},
+			"newrelic_synthetics_monitor_script":    {Tok: makeResource(syntheticsMod, "MonitorScript")},
+			"newrelic_insights_event":               {Tok: makeResource(insightsMod, "Event")},
+			"newrelic_plugins_alert_condition":      {Tok: makeResource(pluginsMod, "AlertCondition")},
+			"newrelic_application_label":            {Tok: makeResource(pluginsMod, "ApplicationLabel")},
```
Comment (mikhailshilkov) [inline resources.go]: This resource is marked as deprecated. Do we need it? https://www.terraform.io/docs/providers/newrelic/r/application_label.html

Comment (mikhailshilkov) [inline resources.go]: Is it on purpose in `plugins`?

File: resources.go [hunk 2]
```diff
@@ -150,30 +150,44 @@ func Provider() tfbridge.ProviderInfo {
...
+			"newrelic_synthetics_label":             {Tok: makeResource(pluginsMod, "SyntheticsLabel")},
+			"newrelic_synthetics_secure_credential": {Tok: makeResource(pluginsMod, "SyntheticsSecureCredential")},
```
Comment (mikhailshilkov) [inline resources.go]: Should these two be in `syntheticsMod`? And maybe the names should omit `Synthetics` then?

File: resources.go [hunk 3]
```diff
@@ -150,30 +150,44 @@ func Provider() tfbridge.ProviderInfo {
...
-			"newrelic_plugin":             {Tok: makeDataSource(pluginsMod, "getPlugin")},
-			"newrelic_plugin_component":   {Tok: makeDataSource(pluginsMod, "getPluginComponent")},
+			"newrelic_alert_channel":                {Tok: makeDataSource(mainMod, "getAlertChannel")},
+			"newrelic_alert_policy":                 {Tok: makeDataSource(mainMod, "getAlertPolicy")},
+			"newrelic_application":                  {Tok: makeDataSource(mainMod, "getApplication")},
+			"newrelic_key_transaction":              {Tok: makeDataSource(mainMod, "getKeyTransaction")},
+			"newrelic_synthetics_monitor":           {Tok: makeDataSource(syntheticsMod, "getMonitor")},
+			"newrelic_plugin":                       {Tok: makeDataSource(pluginsMod, "getPlugin")},
+			"newrelic_plugin_component":             {Tok: makeDataSource(pluginsMod, "getPluginComponent")},
+			"newrelic_synthetics_secure_credential": {Tok: makeDataSource(pluginsMod, "getSyntheticsSecureCredential")},
```
Comment (mikhailshilkov) [inline resources.go]: Also `syntheticsMod`?

File: resources.go (hunk 1 above)
Comment (stack72) [reply resources.go]: Wow - that was just added in the last iteration...

Comment (stack72) [reply resources.go]: Removing this

**Comment by stack72** [comment]
@mikhailshilkov all fixed - nice catches!

**Pull request merged**